ergebnismeldung/src/main/java/com/fb/dlqpattern/ergebnismeldung/service/SendErgebnismeldungService.java

```diff
@@ -0,0 +1,11 @@
+package com.fb.dlqpattern.ergebnismeldung.service;
+
+import com.fb.dlqpattern.ergebnismeldung.rest.SendInfoDTO;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class SendErgebnismeldungService {
+    public void sendErgebnismeldung(SendInfoDTO body) {
+        log.debug("Nachricht erhalten");
```

Comment:
Sehr gut, aber bitte hier **ein bisschen** mehr machen. Zumindest aus DTO ein SendInfo - Objekt mappen/kreieren und ausgeben. (das würde man evtl in die DB speichern). Man könnte auch eine andere Methode hier schreiben, die wenn hier 3 SendInfoDTOs erhalten würde, diese in ein anderes Objekt, dass eine Liste dieser Info enthält, einpacken würde und an den nächsten Service zusenden.

Reply:
soll ich unsere SendInfo klasse als Entity definieren und dann mit einem Repository arbeiten?

Reply:
Ja das wäre gut und eine h2 (in Memory) Datenbank aktivieren, aber das kann in einem weiteren Issue stattfinden. Jetzt lass das so und kannst dazu ein Issue eröffnen.

Reply:
Also wenn das Issue erstellst hast , kannst die Koversation resolven und aus meiner Sicht auch mergen.

Reply:
issue opened